src/components/button_panel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+// import ReactDOM from 'react-dom';
+// import PropTypes from 'prop-types';
+import Buttons from './button';
```

Comment:
- [ ] [OPTIONAL] You are calling a default export Buttons from a file called `button.js` this can be confusing, I recommend you to either change the name of the file or the name of the export.